### ISSUE TITLE
Price update

### ISSUE THREAD

**Issue by hogesonline** [SilverMaddy/ncss-coffeerun#10]
Hey, when I edit a coffee from small to large it doesn't update the price. Cheers.

**Comment by katharosada** [comment]
Yeah, ditto when I had to switch cafe's from a run. Not sure what to do
though, since you can manually set the price and we might not want to wipe
over that.
